attributes/default.rb

```diff
@@ -38,8 +38,8 @@
   # You should store this password in an encrypted data bag item and
   # override in your wrapper. We've set a default password here purely
   # for testing purposes.
-  omnibus['build_user_password'] = if platform_family == 'windows' || platform_family == 'mac_os_x'
-                                     # Passsword must be clear-text on Windows and Mac OS X.
+  omnibus['build_user_password'] = if node['platform_family'] == 'windows' || node['platform_family'] == 'mac_os_x'
```

Comment:
If we can use `platform_family?()` it may be a little cleaner. Eg:` 'get0ntheBus' if platform_family?('windows', 'mac_os_x')`